File: src/Actor.GameHub.Identity.EntityFrameworkCore/IdentityDbContext.cs
using System;
using Microsoft.EntityFrameworkCore;

namespace Actor.GameHub.Identity.EntityFrameworkCore
{
  public class IdentityDbContext : DbContext
  {
    public DbSet<UserEntity> User { get; init; } = null!;

    public IdentityDbContext(DbContextOptions<IdentityDbContext> dbOptions)
      : base(dbOptions)
    {
    }

    protected override void OnModelCreating(ModelBuilder modelBuilder)
    {
      modelBuilder.HasDefaultSchema("Identity");
      
      var userBuilder = modelBuilder.Entity<UserEntity>();
      userBuilder.ToTable("User");
      userBuilder
        .Property(u => u.Username)
        .HasMaxLength(100)
        .IsRequired();
      userBuilder
        .HasIndex(u => u.Username)
        .IsUnique();

      userBuilder.HasData(new UserEntity
      {
        Id = Guid.NewGuid(),
        Username = "lars",
      }, new UserEntity
      {
        Id = Guid.NewGuid(),
        Username = "merten",
      }, new UserEntity
      {
        Id = Guid.NewGuid(),
        Username = "sam",
      }, new UserEntity
      {
        Id = Guid.NewGuid(),
        Username = "uli",
      });

      base.OnModelCreating(modelBuilder);
    }
  }
}

File: fun/Actor.GameHub.Identity.EntityFrameworkCore.FSharp/IdentityDbContext.fs
module Actor.GameHub.Identity.EntityFrameworkCore.IdentityDbContext

open System
open Microsoft.EntityFrameworkCore
open Actor.GameHub.Identity.EntityFrameworkCore.UserEntity
open System.Reflection

type IdentityDbContext(dbOptions: DbContextOptions<IdentityDbContext>) =
    inherit DbContext(dbOptions)

    [<DefaultValue>]
    val mutable private _user: DbSet<UserEntity>

    member this.User
        with get () = this._user
        and set v = this._user <- v

    override this.OnModelCreating modelBuilder =
        modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly())
        |> ignore

        modelBuilder
            .Entity<UserEntity>()
            .HasData(
                { Id = Guid.NewGuid()
                  Username = "lars" },
                { Id = Guid.NewGuid()
                  Username = "merten" },
                { Id = Guid.NewGuid()
                  Username = "sam" },
                { Id = Guid.NewGuid()
                  Username = "uli" }
            )
        |> ignore
